scripts/process/merge_centroids_wu.R

```diff
@@ -1,17 +1,18 @@
 process.merge_centroids_wu <- function(viz) {
   deps <- readDepends(viz)
   wu_data_15_simple <- deps[["wu_data_15_simple"]]
-  topo <- geojsonio::topojson_read(as.viz(viz$depends$topo)$location, stringsAsFactors=FALSE) # bypass readDepends for topojson
+  topo <- rgdal::readOGR(as.viz(viz$depends$topo)$location, layer=viz$process_args$layer, drop_unsupported_fields=TRUE, stringsAsFactors=FALSE) # bypass readDepends for topojson
   
-  # merge datasets and keep a minimal set of columns
+    # merge datasets and keep a minimal set of columns
   county_data <- topo@data %>%
     left_join(wu_data_15_simple, by=c('GEOID'='FIPS')) %>%
     select(GEOID, STATE_ABBV, COUNTY, countypop:industrial) %>% 
     rowwise() %>% 
     mutate(other = total - sum(thermoelectric, publicsupply, irrigation, industrial))
   
+  # put the result back into the topojson
   topo@data <- county_data
   
-  # write to file
-  geojsonio::topojson_write(topo, file=viz[['location']])
+  # write to file. don't use geojsonio::topojson_write because we can't specify two layers [yet; geojsonio is growing fast]
+  geojsonio::topojson_write(topo, file=viz[['location']], overwrite=TRUE, object_name=viz$process_args$layer)
```

Comment:
so your comment on the line above this one says "don't use" but we are still using `geojsonio::topojson_write`. Is there an alternative we should be using here?

Reply:
oh, right. i tried several options, but i think this one is the best. i was trying to get states and counties into the same file, with water use data, but ultimately had to give up. i tried reading and writing using jsonlite, but that was corrupting my topojson in ways I couldn't find or fix. and as noted in that comment (which i'll delete), with the option i chose i've had to go back to having separate files for states and counties - a shame, but also should allow us to load the states before the counties for faster time to viewable content.